packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -201,11 +209,13 @@ Media.createMediaConnection = (
       },
     },
     {
+      // TODO: RoapMediaConnection is not ready to use stream classes yet, so we pass the raw MediaStreamTrack for now
```

Comment:
it would be better to reference a jira in any TODOs that we put in the code

Reply:
I'll create a Jira for it and add. Thanks

@marcin-bazyl / @sreenara - We should actually create this story to take up after the v3 release. Is that correct?

Reply:
@mkesavan13 yes we don't have a plan to do that right now. We can come back to it at a later time if there is a use-case for it.

Reply:
@sreenara so there is no plan to move calling SDK to use webrtc-core tracks at all?

Reply:
ok, great, but if you are planning to do it, surely you have a jira :-) can you reference it here in the todo comment?

Reply:
Created [SPARK-460530](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-460530) to do this in the future once we get some bandwidth.

Reply:
thanks, @mkesavan13 can you reference [SPARK-460530](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-460530) in this todo comment, please?

Reply:
Done in #3053